### PR TITLE
Allow users to skip the URL stability check themselves

### PR DESCRIPTION
**Pull request by Wyverald**
by adding a comment saying `@bazel-io skip_check unstable_url`.

### CODE REVIEW

**Review by fmeum — APPROVED** [review]
I don't know the GitHub API well enough to catch security issues, but this looks good to me with respect to its functionality.

File: tools/bcr_validation.py [hunk 1]
```diff
@@ -179,10 +179,13 @@ def verify_source_archive_url_stability(self, module_name, version):
...
                 BcrValidationResult.FAILED,
                 f"{module_name}@{version} is using an unstable source url: `{source_url}`.\n"
-                + "You should use a release archive URL in the format of "
+                + "If at all possible, you should use a release archive URL in the format of "
                 + "`https://github.com/<ORGANIZATION>/<REPO>/releases/download/<version>/<name>.tar.gz` "
                 + "to ensure the archive checksum stability.\n"
-                + "See https://blog.bazel.build/2023/02/15/github-archive-checksum.html for more context.",
+                + "See https://blog.bazel.build/2023/02/15/github-archive-checksum.html for more context.\n"
+                + "If no release archives are available, please add a comment to your BCR PR with the text\n"
+                + "    @bazel-io unstable_ack\n"
```
Comment (meteorcloudy) [inline tools/bcr_validation.py:187]: Can we use `@bazel-io skip url_check` instead? I can think of using the same way for skipping other checks I want to add later.

File: .github/workflows/comment_bot.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    permissions:
+      issues: write
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+        with:
+          egress-policy: audit
+
+      - name: Run bot
+        uses: bazelbuild/continuous-integration/actions/release-helper@728620684433bc54e4bb77613fa79977ba8da106 # master
```
Comment (meteorcloudy) [inline .github/workflows/comment_bot.yml:21]: Feels weird we are mixing this with the release-helper, can we separate those?

File: tools/bcr_validation.py (hunk 1 above)
Comment (Wyverald) [reply tools/bcr_validation.py]: Done.

File: .github/workflows/comment_bot.yml (hunk 2 above)
Comment (Wyverald) [reply .github/workflows/comment_bot.yml]: Done.

**Review by meteorcloudy — APPROVED** [review]
Awesome, thanks!

**Comment by meteorcloudy** [comment]
FYI @wep21 @Vertexwahn

**Pull request merged**